.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: 17
```

Comment:
(non-blocking) Any reason for going to 17 if you still stay on 11 for the code itself?

Reply:
Yes; the sonarqube Gradle plugin requires Java 17 to work.

So the build itself needs to run with Java 17. The code itself is compiled with Java 11, so you can still run Gradle with an older Java version (e.g. for older Alfresco builds)